RecoTracker/MkFitCore/src/PropagationMPlex.cc

```diff
@@ -488,7 +488,19 @@ namespace mkfit {
     errorProp.setVal(0.f);
     outFailFlag.setVal(0.f);
 
+    //helixAtRFromIterativeCCS_impl_new(inPar, inChg, msRad, outPar, errorProp, outFailFlag, 0, NN, N_proc, pflags);
+    helixAtRFromIterativeCCS_impl(inPar, inChg, msRad, outPar, errorProp, outFailFlag, 0, NN, N_proc, pflags);
+    /*
```

Comment:
Is there a good reason to keep all this commented code?

Reply:
I have removed this (and the much longer implementation in the .icc file) for now and left a comment below  with a reference to specific commit in case this needs to be revisited.